engineering_tools/carma_script_extensions/cloud-sim

```diff
@@ -358,6 +351,211 @@ cloud-sim__status() {
     aws cloudformation describe-stack-resources --stack-name "${STACK_NAME}"
 }
 
+# Function to create the VPC, internet gateway, and route table
+cloud-sim__init() {
+
+    if [ ! command -v "aws" &> /dev/null ]; then
+        echo "aws cli is not installed. Attempting to install. This requires sudo."
+        
+        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+        unzip awscliv2.zip
+        sudo ./aws/install
+
+        echo "NOTE: You need to manually configure the aws cli for your user access key. Refer to this video for instructions https://www.youtube.com/watch?v=Rp-A84oh4G8"
+        exit 1
+    fi
+    
+    # Check if resources already exist - reset if all don't exist
+    local vpc_success=0
+    local igw_success=0
+    local route_table_success=0
+
+    # Get resource status - query_resource_status defines VPC, Internet Gateway and Route Table ID
+
+    if query_resource_status; then
+        echo "Xil Cloud resources already exist. No change required. Run start to bring up stack.
```

Comment:
It's not a huge deal, but I would recommend where possible we change "XIL" to "CDASim" here, XIL is the project name, but the product is CDASim.

Reply:
Also would be good to change "start" to "carma cloud-sim start" so it's unambiguous, for users that haven't done this before.